app.py

```diff
@@ -66,6 +69,35 @@ def edit(id):
     return jsonify({'slugs': result})
 
 
+@app.route('/ssl', methods=['POST'])
+def add_ssl():
+    """Receive an SSL certificate"""
+    query = request.get_json()
+    name = query.get('name')
+    cert = query.get('cert')
+    key = query.get('key')
+    with open('{0}/{1}.crt'.format(SSL_PATH, name), 'w') as f:
+        f.write(cert)
+    with open('{0}/{1}.key'.format(SSL_PATH, name), 'w') as f:
+        f.write(key)
+
+
+@app.route('/ssl/<name>', methods=['DELETE'])
+def delete_ssl(name):
+    """Delete an SSL certificate"""
+    # Resolve the path
+    key_path = abspath(
+        realpath(
+            '{0}/{1}.key'.format(SSL_PATH, name)))
+    cert_path = abspath(
+        realpath(
+            '{0}/{1}.key'.format(SSL_PATH, name)))
+    if not key_path.startswith(SSL_PATH) or \
+       not cert_path.startswith(SSL_PATH):
+        return False
+    remove(key_path)
```

Comment:
No operation should be done in this file. It's supposed to be used only to clear incoming data or format outgoing data. In the Nginx class, a ssl_delete or delete_ssl (or something like this) would best. Or a new SSL class ?